pyproject.toml

```diff
@@ -16,6 +16,7 @@ for use in data storage and communication applications."""
 readme =  "README.rst"
 dependencies = [
     "numpy>=1.7",
+    "crc32c>=2.7",
```

Comment:
I don't have any strong feelings either way, but is it worth making this an optional extra instead of being in the main dependencies?

Reply:
I made it an optional dep. Seems like making it a core dependency later is easier than the other way around.